File: and-bible/app/src/main/java/net/bible/service/device/speak/event/SpeakProggressEvent.java
package net.bible.service.device.speak.event;

import org.crosswire.jsword.book.Book;
import org.crosswire.jsword.passage.Key;

public class SpeakProggressEvent {
    public Book book;
    public Key key;
    public SpeakProggressEvent(Book book, Key key) {
        this.key = key;
        this.book = book;
    }
}

File: and-bible/app/src/main/java/net/bible/service/device/speak/event/SpeakProggressEvent.kt
package net.bible.service.device.speak.event

import net.bible.service.device.speak.SpeakCommand
import org.crosswire.jsword.book.Book
import org.crosswire.jsword.passage.Key

data class SpeakProggressEvent(val book: Book, val key: Key, val synchronize: Boolean, val speakCommand: SpeakCommand)